[doc] Check in an example config

donaldong opened the pull request:
### Summary
This adds an example configuration, modified from the ones we have in traject, as a reference.

donaldong commented:
update

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redis-memo/compare/6c9a464991eab995cd063ada7783aa0bc257ea52..0cd8f7d61acae5d8e6b4dd08bf508f3c7f383b01).

donaldong commented:
Remove reference to traject

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redis-memo/compare/33fff8633b5e31d1ad9340003da9930f468953a0..d0c0613922847fb448cc3ee6a4b2af7e921d1f01).

donaldong commented:
fix rubocop

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redis-memo/compare/b212d646f188dca79a26fd47541877491fcb140c..6202a845da3dac4f8b910d54ccf3dfa3a6b3cd82).

Dingying0410 commented on docs/examples/config.rb:
```diff
@@ -0,0 +1,78 @@
+    }
+  end
+
+  config.cache_validation_sample_percentage = ENV['REDIS_MEMO_CACHE_VALIDATION_SAMPLE_PERCENTAGE']&.to_i
+  config.cache_out_of_date_handler = proc do |ref, method_id, args, cached_result, fresh_result|
```
```suggestion
  config.cache_validation_sample_percentage = ENV['REDIS_MEMO_CACHE_VALIDATION_SAMPLE_PERCENTAGE']&.to_i

  config.cache_out_of_date_handler = proc do |ref, method_id, args, cached_result, fresh_result|
```

line seperation

Dingying0410 commented on docs/examples/config.rb:
```diff
@@ -0,0 +1,78 @@
+    }
+  end
+
+  config.cache_validation_sample_percentage = ENV['REDIS_MEMO_CACHE_VALIDATION_SAMPLE_PERCENTAGE']&.to_i
```
can we add a comment to say this is a value between 0 to 100?  (not 0  to 1)

Dingying0410 commented on docs/examples/config.rb:
```diff
@@ -0,0 +1,78 @@
+  # Static references: Those references are fixed when booting the app
+  config.expires_in = 3.hours
+
+  # On non-heroku environments, clear the cache when starting a new Rails
+  # server process
+  config.global_cache_key_version = ENV['HEROKU_SLUG_COMMIT']
```
Hmm it feels blurred to me when reading non heroku environments with the `HEROKU_SLUG_COMMIT` Maybe also add a comment here for what to do in Heroku environments, and the example we provide is for Heroku env?.

Dingying0410 reviewed (approved):
Awesome, lgtm!

katyho reviewed (approved):
Maybe we can also provide a link to the rdoc description of each config value as a comment

donaldong commented:
update comments

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redis-memo/compare/47cc8256936e06e9af732f7d79cde264594915d5..08a53186798b8bd1709ecea94ec3e6f3da9d9c04).

donaldong commented:
remove an extra char

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redis-memo/compare/3677a3b4c7209472f18bf433fb7162364444e465..7681b50c97c8d4cae26e5ca92c5641d9b20c4fbd).
